pallets/msa/src/lib.rs

```diff
@@ -848,11 +867,48 @@ pub mod pallet {
 					Self::deposit_event(Event::MsaRetired { msa_id });
 				},
 				None => {
-					log_err!("SignedExtension did not catch invalid MSA for account {:?}, ", who);
+					log::error!(
+						"SignedExtension did not catch invalid MSA for account {:?}, ",
+						who
+					);
 				},
 			}
 			Ok(Some(T::WeightInfo::retire_msa(num_deletions)).into())
 		}
+
+		/// Request to be a provider
+		#[pallet::call_index(11)]
+		#[pallet::weight(1000)]
+		pub fn request_to_be_provider(
```

Comment:
Should the request be the one that is being called through governance? Once the request is accepted then the provider fills in their info?